### PR TITLE
Add script to update tables with UTF character set

### PR DESCRIPTION
**Pull request by ehunter-usgs**
This is part of a gitlab issue for puppet (issue-10) to update UTF-8 encoding

### CODE REVIEW

**Comment by jmfee-usgs** [comment]
If using the default collation here, which is probably more future proof since mysql said utf8 will eventually be removed and replaced with utf8mb4, will you also update the other scripts to only specify the character set?

File: src/lib/sql/fdsnws/getEventSummary.sql [hunk 1]
```diff
@@ -8,7 +8,7 @@ CREATE PROCEDURE getEventSummary(
   OUT out_event_type TEXT,
   OUT out_azimuthal_gap DOUBLE,
   OUT out_magnitude_type TEXT,
-  OUT out_region TEXT,
+  OUT out_region TEXT CHARACTER SET utf8,
```
Comment (jmfee-usgs) [inline src/lib/sql/fdsnws/getEventSummary.sql:11]: I think this change should be to all columns.

Pretty sure utf8 would become the default if you:
```
SET NAMES utf8;
-- recreate stored procedures
```

File: src/lib/sql/utf8-update.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ALTER TABLE focalMechanismSummary CONVERT TO CHARACTER SET utf8;
+ALTER TABLE momentTensorSummary CONVERT TO CHARACTER SET utf8;
+ALTER TABLE originSummary CONVERT TO CHARACTER SET utf8;
+ALTER TABLE productSummary CONVERT TO CHARACTER SET utf8;
+ALTER TABLE productSummaryEventStatus CONVERT TO CHARACTER SET utf8;
+ALTER TABLE productSummaryLink CONVERT TO CHARACTER SET utf8;
+ALTER TABLE productSummaryProperty CONVERT TO CHARACTER SET utf8;
+
+## Stored procedure region/title/place character set
+SOURCE fdsnws/getEventSummary.sql;
```
Comment (jmfee-usgs) [inline src/lib/sql/utf8-update.sql:26]: This works okay when running from local directory structure, but makes for 4 files to copy to a remote server...

Comment (Capprin) [reply src/lib/sql/utf8-update.sql:26]: all of these files already exist in the built event-ws application; is it a problem to include them like this?

Comment (ehunter-usgs) [reply src/lib/sql/utf8-update.sql:28]: I agree with @Capprin, these sql scripts are already in place on the server. Creating another copy would increase the likelihood that those procedures could fall out of sync if they were updated in the future.

File: src/lib/sql/fdsnws/getEventSummary.sql (hunk 1 above)
Comment (ehunter-usgs) [reply src/lib/sql/fdsnws/getEventSummary.sql]: Updated.

**Pull request merged**